x/erc20/types/utils_test.go

```diff
@@ -25,3 +26,83 @@ func TestSanitizeERC20Name(t *testing.T) {
 		require.Equal(t, tc.expErc20Name, name, tc.name)
 	}
 }
+
+func TestEqualMetadata(t *testing.T) {
```

Comment:
can you refactor this test to be a table test?

Reply:
> can you refactor this test to be a table test?

Finished.